src/config/texts.js

```diff
@@ -2,6 +2,10 @@ import appJson from '../../app.json';
 
 export const texts = {
   accessibilityLabels: {
+    checkbox: {
+      active: 'Ausgewählt',
+      inactive: 'Nicht ausgewählt'
```

Comment:
let's write it lowercased as these are not nouns in German

Reply:
written in lower case
https://github.com/smart-village-solutions/smart-village-app-app/commit/f425b1ee2acfc59d90979d9e08ec3fa64b692c06